src/commonmark/en/content/user/using-the-capture-app.md

```diff
@@ -429,48 +429,90 @@ The Capture app doesn't support tracker programs yet, but the tracker programs a
 
    ![](resources/images/capture_app/search-by-attributes-find-button.png)
 
-These steps will take you to the search page. There, given the configuration your organisation has in the maintenance app you will see the different attributes you can search with. An example of how this looks like is the following.
+    These steps will take you to the search page. There, based on the configuration of your organisation, will see the different attributes you can search with. An example of how this looks is the following.
 
-![](resources/images/capture_app/search-by-attributes-on-scope-program-overview-0.png)
+    ![](resources/images/capture_app/search-by-attributes-on-scope-program-overview-0.png)
 
-To execute a search now, 
+    To execute a search now: 
 
 1. Fill in the attributes you want to search with.
 
-2. Clcik the Search by attributes button.
+2. Click the **Search by attributes** button.
 
    ![](resources/images/capture_app/search-by-attributes-on-scope-program-overview-1.png)
+   
+3. The results of the search will be displayed as follows.
+
+   ![](resources/images/capture_app/search-by-attributes-on-scope-program-overview-2.png)
+    
+   In this list you can see the entries that match your search. For each entry you can have a total of three options. 
+    
+   a. You can choose to view the users dashboard by clicking the "View dashboard" button
```

Comment:
I think "user" is the wrong term here. The TE might not be a person. On the other hand "tracked entity" or "entity" are too technical. "Patient" might be more applicable than user, but it is still only relevant to people, and also only in the health domain.
This might be a question for @Bekkalizer or @Wonfrost